### ISSUE TITLE
Make lbd server running from base4 public

### ISSUE THREAD

**Issue by travula** [vlead/vlabs-learn-by-doing#2]
@sivashankerN 
@BalammaBoya 

Please give the lbd (learn by Design) server running from base4 public access?
The domain name should be lbd.vlabs.ac.in

**Comment by travula** [comment]
@sivashankerN
@BalammaBoya

Please do this by tonight (23/7/2017)

**Comment by travula** [comment]
​Shiva,

Please look at the issue
<https://github.com/vlead/vlabs-learn-by-doing/issues/2>. ​ I learnt from
Raghupathi that the current installation on base4 is in working condition.
Please confirm that this installation is tested and provide a public access
to this machine.

- Thirumal

**Comment by travula** [comment]
Sure Thirumal,

On Mon, Jul 24, 2017 at 9:29 AM, Thirumal Ravula <travula@vlabs.ac.in>
wrote:

> ​Shiva,
>
> Please look at the issue
> <https://github.com/vlead/vlabs-learn-by-doing/issues/2>. ​ I learnt from
> Raghupathi that the current installation on base4 is in working condition.
> Please confirm that this installation is tested and provide a public access
> to this machine.
>
> - Thirumal
>
> On Jul 24, 2017 8:06 AM, "Siva Shanker Narimalla" <shankar@vlabs.ac.in>
> wrote:
>
>> Dear Professor,
>>
>> There is a problem with auto deployment. We are to deploy LbD using ADS
>> in our staging/testing environment with proper links.
>>
>> After hosting this LbD using ADS we are unable to see the web content.
>>
>> Regarding Videos: Sripathi and  Balamma working on this taks. They will
>> give us more info on this.
>>
>> As of now we have hosted LbD in our one base machine and it is working
>> fine. It has updated links(lbd.vlabs.ac.in)  too.
>>
>> If we are okay publicing this hosted lab from IIITH infrastructure, It
>> will be done today end of the day.
>>
>>
>>
>>
>> On 24 Jul 2017 7:01 a.m., "Venkatesh Choppella" <
>> venkatesh.choppella@iiit.ac.in> wrote:
>>
>>> Hello Siva, Balamma, Thirumal,
>>>
>>> I need confirmation that LbD has been hosted successfully with all
>>> links properly resolved.
>>>
>>> Currently, lbd.vlabs.ac.in shows up nothing.
>>>
>>>
>>> Sincerely,
>>>
>>> Venkatesh Choppella, PhD.
>>> Associate Professor
>>> IIIT Hyderabad
>>> Gachibowli, Hyderabad 500 032
>>> Ph: +91-965-274-0281
>>>
>>


-- 
Thanks & Regards,
Siva Shanker Narimalla,
Project Engineer
VLEAD, IIIT Hyderabad.

**Comment by sivashankerN** [comment]
@travula Yes Thirumal, As Raghupathi said, we have working LbD lab on base4 machine ( not from cluster).
I did not see this issue yesterday otherwise I would have done this yesterday night only.

@kraghupathi and @BalammaBoya wants to do final testing.
Give us one hour time, we will make working LbD lab publicly accessible. Name will be lbd.vlabs.ac.in

**Comment by kraghupathi** [comment]
The LbD server is now publicly accessible and it is tested from outside IIIT-H whether its working or not (working properly).

Please following the links of LbD server.
1. [LbD-wiki](http://lbd.vlabs.ac.in/wiki/)
2. [LbD-moodle](http://lbd.vlabs.ac.in/moodle/)

**Comment by BalammaBoya** [comment]
Dear all,

I have cross check course by course about the links. Please find the
following details about links which are not working in LbD course according
to course wise.

*1. Operating System*
Illustration and playground links are not working in all units/modules.
They are serving from the following link.
http://lbd.vlabs.ac.in/MHRDAnimations/OS/

*2. Solid State Electronics Devices*
Illustrations links are not working in all units/modules. They are serving
from the following link.
http://lbd.vlabs.ac.in/MHRDAnimations/SSED/ssed_corrected/

Assessment links also not working

*3. Algorithms*
Playground links are not working in all units/modules. They are serving
from the following link
Algorithms%20-%20Divide%20and%20conquer.swf

*4. Digital Systems Design using Verilog*
Playground links are not working in all units/modules. They are serving
from the following link
http://lbd.vlabs.ac.in/MHRDAnimations/DSD/


*5. Data Mining and Data Warehousing*
Playground links are not working in all units/modules. They are serving
from the following link
http://lbd.vlabs.ac.in/animation/dmdw/

*6. Embedded Systems*
Playground links are not working in all units/modules. They are serving
from the following link
http://lbd.vlabs.ac.in/MHRDAnimations/EmbeddedSystems/
Introduction to e-governance
videos links are not working

*7. Social Networking*
Videos links are not working in all the units/modules
http://lbd.vlabs.ac.in/pdf_wiki/PDF Wiki Courses_Version1.1/Social
Networking/

*8. Analog Design*
Videos links are redirecting into ftp server in all the units/modules. They
are serving from the following link
ftp://10.3.0.63/CIT_VIDEOS/mhrd/Analog/Analog%20Filters/

*9. Digital Logic Design*
Playground links are not working in all units/modules. They are serving
from the following link
http://lbd.vlabs.ac.in/virtuallabs/digital-logic-design-iiith/

*10.* *Introduction to e-governance*
Videos links are not working in all the units/modules. They are serving
from the following link
http://lbd.vlabs.ac.in/pdf_wiki


*11. Advanced Topics in Text and Web Mining*
This course links are working fine

Illustration and Playground sources files are available in wiki
<http://lbd.vlabs.ac.in/wiki/index.php/Main_Page> we can take it from
there.


Thank you
With Best Regards,
Balamma

**Comment by BalammaBoya** [comment]
We have fixed the following courses links and documented them.

Raghupathi copied the following course source files which available on ftp server and pdf_wiki directory. I have fixed them on the moodle.

7. Social Networking
8. Analog Design
9. Digital Logic Design
10. Introduction to e-governance

The following courses links are not working because in the enhanceedu production server, "MHRDAnimations and animation" directory/folder is missing which are serving the following courses "Playground and Illustrations". [link](http://lbd.vlabs.ac.in/MHRDAnimations/)

1. Operating System
2. Solid State Electronics Devices
3. Algorithms
4. Digital Systems Design using Verilog
5. Data Mining and Data Warehousing
6. Embedded Systems

The following course already working fine
11. Advanced Topics in Text and Web Mining



[lbdcourses-links_document.docx](https://github.com/vlead/vlabs-learn-by-doing/files/1173150/lbdcourses-links_document.docx)

**Comment by BalammaBoya** [comment]
[lbd.vlabs.ac.in/moodle](http://lbd.vlabs.ac.in/moodle) all links are working fine **except that missing directories of "MHRDAnimations and animation" in the enhanceedu production server.**

**Comment by BalammaBoya** [comment]
We found the "MHRDAnimations" directory/folder from the backups with the help of Nadeem (enhanceedu previous system admin). 
Raghupathi copied the files into production. I have fixed them on moodle. Now all links are working fine.

**Comment by kraghupathi** [comment]
Documentation found at [here](https://github.com/vlead/vlabs-learn-by-doing/blob/master/src/index.org) for VLABS LbD.
